src/main/java/org/radarcns/util/EmailSenders.java

```diff
@@ -0,0 +1,85 @@
+package org.radarcns.util;
+
+import org.radarcns.config.MonitorConfig;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * Class to store {@link EmailSender} associated with each project.
+ */
+public class EmailSenders {
```

Comment:
Why not just expose a map of `<string, emailsender>`?

Reply:
For future if need to add more conditions to send email instead of just a project. 
Also makes the code easy to read and understand. eg - `getEmailSender()` instead of `get()`.
If its not a good practice then i can update it to expose a map

Reply:
Having a separate class is fine, if only the functionality that is going to be used is included.

Reply:
Okay got it. I will update it based on comments. Thanks